resources/Dockerfile.phan

```diff
@@ -0,0 +1,11 @@
+FROM php:7.2-buster
+RUN apt-get -y update && apt-get -y install git zip && \
+curl -sS https://getcomposer.org/installer | php && \
+mv composer.phar /usr/local/bin/composer && \
+chmod +x /usr/local/bin/composer && \
+pecl install ast-1.0.4 && \
```

Comment:
What is the ast extension for?

Reply:
It was a requirement for phan.

Reply:
From the `composer install` output:
`phan/phan suggests installing ext-ast (Needed for parsing ASTs (unless --use-fallback-parser is used). 1.0.1+ is needed.)`